.github/workflows/basic-continuous-integration.yml

```diff
@@ -11,6 +11,12 @@ jobs:
       - name: Checkout Code
         uses: actions/checkout@v2
 
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.2
+          ini-values: assert.exception=1, zend.assertions=1
```

Comment:
What's the purpose of these values being set?

Reply:
They are used for assertions and to throw exceptions if they fail.
Maybe they’re not as important here, though, and we use the same config in different projects as well.